FEWD/Week03/lesson05/homework/week3-relaxr-part2/css/style.css

```diff
@@ -0,0 +1,211 @@
+/*
+Project Name: Blank Template
+Client: Your Client
+Author: Your Name
+Developer @GA in NYC
+*/
+
+
+/******************************************
+/* SETUP
+/*******************************************/
+
+/* Box Model Hack */
+* {
+ box-sizing: border-box; /* IE */
+}
+
+/* Clear fix hack */
+.clearfix:before, .clearfix:after {
+  content: "";
+  display: table;
+}
+
+.clearfix:after{
+  clear: both;
+}
+
+.alignright {
+  float: right;
+  padding: 0 0 10px 10px; /* note the padding around a right floated image */
+}
+
+.alignleft {
+  float: left;
+  padding: 0 10px 10px 0; /* note the padding around a left floated image */
+}
+
+/******************************************
+/* BASE STYLES
+/*******************************************/
+
+body {
+  background-color: #f0efef;
+  color: #000;
+  font-size: 12px;
+  line-height: 1.4;
+  font-family: 'Open Sans', Helvetica, Arial, sans-serif;
+  width: 100%;
+}
+
+
+/******************************************
+/* LAYOUT
+/*******************************************/
+
+header {
+  background-color: #033048;
+  height: 80px;
+}
+
+h1 {
+  font-size: 40px;
+  color: white;
+  font-weight: 700;
+  padding: 10px 0 0 70px;
+  float: left;
+}
+
+nav {
+  float: right;
+  padding-right: 75px;
+}
+
+nav li {
+  font-size: 18px;
+  display: inline-block;
+  padding-right: 30px;
+  padding-top: 28px;
+}
+
+nav li a {
+  text-decoration: none;
+  color: #f9e42e;
+}
+
+main {
+  background-color: #f0efef;
+  width: 80%;
```

Comment:
Given that you've limited the `width` of the two children of `<main>` (which are are `<section>` and `<aside>` in pixel values, and the fact that the README specifies a gutter/gap of `20px` between the two columns, you may want to consider using a `width` in `px` here.

Since `<section>` is `700xp` and `<aside>` is 240px and you're meant to have 20px between them, you'll want:

```
main {
  width: 960px;
  margin: 0 auto;
}
```
The result should look like this:

![image](https://cloud.githubusercontent.com/assets/5670279/17485277/19d6fe8c-5d85-11e6-83ab-100478ce32ec.png)

... which looks pretty nice, a more "tucked in" design, with less of a gap between the two columns. Additionally, you'll notice that in my proposed CSS for `<main>` above, I've left out `background-color` and that's because you've already set that same background-color on the `<body>` so there's no need to repeat it here